### ISSUE TITLE
Error while running 'docker-compose build'

### ISSUE THREAD

**Issue by rafaykh90** [jupyter-incubator/sparkmagic#586]
I cloned the repo and trying to create the Docker image using the `docker-compose build` command. After some time, it fails with error: 

```
[ERROR] Failed to execute goal net.alchim31.maven:scala-maven-plugin:3.2.2:compile (scala-compile-first) on project spark-yarn_2.11: Execution scala-compile-first of goal net.alchim31.maven:scala-maven-plugin:3.2.2:compile failed. CompileFailed -> [Help 1]
[ERROR] Failed to execute goal net.alchim31.maven:scala-maven-plugin:3.2.2:compile (scala-compile-first) on project spark-graphx_2.11: Execution scala-compile-first of goal net.alchim31.maven:scala-maven-plugin:3.2.2:compile failed. CompileFailed -> [Help 1]
[ERROR] Failed to execute goal net.alchim31.maven:scala-maven-plugin:3.2.2:compile (scala-compile-first) on project spark-streaming_2.11: Execution scala-compile-first of goal net.alchim31.maven:scala-maven-plugin:3.2.2:compile failed. CompileFailed -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :spark-yarn_2.11
ERROR: Service 'spark' failed to build: The command '/bin/sh -c mkdir -p /apps/build &&   cd /apps/build &&   git clone https://github.com/apache/spark.git spark &&   cd $SPARK_BUILD_PATH &&   git checkout v$SPARK_BUILD_VERSION &&   dev/make-distribution.sh --name spark-$SPARK_BUILD_VERSION -Phive -Phive-thriftserver -Pyarn &&   cp -r /apps/build/spark/dist $SPARK_HOME &&   rm -rf $SPARK_BUILD_PATH' returned a non-zero code: 1
```

The error seems to be in the `Dockerfile.spark` file.

Can someone help me with this? 

Thanks.

**Comment by devender-yadav** [comment]
@rafaykh90 check related conversation - https://github.com/jupyter-incubator/sparkmagic/issues/551

**Comment by devstein** [comment]
Closed by #756
